cclib/scripts/ccframe.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Script for writing data tables from computational chemistry files."""
+
+import sys
+import os.path
+import argparse
+
+try:
+    import pandas as pd
+    _has_pandas = True
+except ImportError:
+    _has_pandas = False
+
+from cclib.io import ccopen
+from cclib.io import ccframe
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-O', '--outputdest',
+                        help=('the output document to write, including an '
+                              'extension supported by pandas '
+                              '(csv, hdf/hdf5, json/cjson, pickle/pkl, xlsx)'))
+    parser.add_argument('compchemlogfile',
+                        nargs='+',
```

Comment:
Minor nitpick: if this is one or more output files, shouldn't the variable be called `compchemlogfiles`?

Reply:
edde255 changed the variable to `compchemlogfiles`, but kept the singular form in the help message:

    usage: ccframe.py [-h] [-O OUTPUT] [--identifier IDENTIFIER]
                  compchemlogfile [compchemlogfile ...]
    (...)